pkg/util/consts.go

```diff
@@ -6,6 +6,7 @@ const (
 	MachineSpecifiedGlobalConfigNamespace = "openshift-config-managed"
 	TargetNamespace                       = "openshift-controller-manager"
 	OperatorNamespace                     = "openshift-controller-manager-operator"
+	InfraNamespace                        = "openshift-infra"
```

Comment:
As an alternative to my comment at https://github.com/openshift/cluster-openshift-controller-manager-operator/pull/216/commits/09eb355f0d0a7866c17b28f5a340936a7326eeb5#diff-0d623dfd885adb20f991bda4c2453aebd732ca6dbb4d1d4be6e79805c3b48de6R57 we could add a constant here named something like `ClusterScoped` that is set to the emtpy string ""